src/test/kotlin/no/fdk/concept_catalog/utils/TestUtils.kt

```diff
@@ -211,3 +215,8 @@ private fun BegrepsRelasjon.mapDBO(): org.bson.Document =
         .append("beskrivelse", beskrivelse)
         .append("inndelingskriterium", inndelingskriterium)
         .append("relatertBegrep", relatertBegrep)
+
+private fun Kontaktpunkt.mapDBO(): org.bson.Document =
+    org.bson.Document()
+        .append("harTelefon", harTelefon)
+        .append("harEpost", harEpost)
```

Comment:
\n @ EOF (konvensjon i git)
```suggestion
        .append("harEpost", harEpost)

```

Reply:
👍